portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,7 +43,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // Fetch the max number of comments from the datastore
     PreparedQuery commentEntities = DEFAULT_DATASTORE_SERVICE.prepare(new Query("Comment"));
-
+    
```

Comment:
tab not need?

I guess you could use some autoformat tools so that you don't need to care the format anymore~